node-export-server/server.js

```diff
@@ -275,23 +341,26 @@ function doExport(req, res, hash, input) {
         //handle different flavors of bbox param
         var bbox_param = 'convert.bounds';
         var bbox = exports.validateBbox(req.query.bbox);
-
+        var poly = exports.validatePoly(req.query.poly);
         if (input.substring(0,4) === 'http') {
             var cert_param = '';
             if (input.substring(0,5) === 'https' && fs.existsSync(appDir + 'key.pem') && fs.existsSync(appDir + 'cert.pem')) {
                 cert_param = '--private-key=' + appDir + 'key.pem --certificate=' + appDir + 'cert.pem';
             }
             temp_file = outDir + '.osm';
-            command += 'wget -q ' + cert_param + ' -O ' + temp_file + ' ' + input + '?bbox=' + bbox + ' && ';
+            command += 'wget -q ' + cert_param + ' -O ' + temp_file + ' ' + input + '?bbox=' + (poly ? exports.polyToBbox(poly) : bbox) + ' && ';
             input = temp_file;
             //bbox not valid with osm file input source
             bbox = null;
         }
 
         //create command and run
         command += 'hoot convert -C NodeExport.conf';
+        var bboxOption = '';
```

Comment:
should we return an error if both are present instead of just giving poly precedent

Reply:
I think it could go either way.  I'm fine with precedence if you are, if you want to make it strict and disallow both that is cool too.